src/main/java/io/vertx/spi/cluster/hazelcast/impl/HazelcastInternalAsyncMap.java

```diff
@@ -50,7 +50,7 @@ public void put(K k, V v, Handler<AsyncResult<Void>> completionHandler) {
     K kk = convertParam(k);
     V vv = convertParam(v);
     executeAsyncVoid(
-            (ICompletableFuture<Void>)map.putAsync(kk, vv),
+            map.setAsync(kk, vv),
```

Comment:
Since the old value returned by `putAsync` is not used, we can use `setAsync` instead and avoid the cost of additional deserialization.